src/Dotnet.Script.Core/ScriptCompiler.cs

```diff
@@ -86,10 +86,12 @@ public virtual ScriptOptions CreateScriptOptions(ScriptContext context)
             var opts = CreateScriptOptions(context);
 
             var runtimeId = RuntimeEnvironment.GetRuntimeIdentifier();
-            var inheritedAssemblyNames = DependencyContext.Default.GetRuntimeAssemblyNames(runtimeId).Where(x =>
-                x.FullName.ToLowerInvariant().StartsWith("system.") ||
-                x.FullName.ToLowerInvariant().StartsWith("microsoft.codeanalysis") ||
-                x.FullName.ToLowerInvariant().StartsWith("mscorlib"));
+            var inheritedAssemblyNames =
+                from an in DependencyContext.Default.GetRuntimeAssemblyNames(runtimeId)
```

Comment:
Okay, I've reverted to using the method chain style though I'm still curious to know why.

PS Don't mind the tongue-in-cheek commit message for 355097881c9ab780a0f633beda20744dd810d41a; I'm assuming you'll squash-merge and retain the commit message from 5f4f125d5bfcd66881626d6d1ff06f9a49bdabd0 but up to you in the end.